Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -0,0 +1,152 @@
+import defaultValue from "../../Core/defaultValue.js";
+import GltfLoader from "../GltfLoader.js";
+import ModelExperimentalUtility from "./ModelExperimentalUtility.js";
+import ModelExperimentalSceneGraph from "./ModelExperimentalSceneGraph.js";
+import Resource from "../../Core/Resource.js";
+import RuntimeError from "../../Core/RuntimeError.js";
+import when from "../../ThirdParty/when.js";
+
+/**
+ * A 3D model based on glTF, the runtime asset format for WebGL. This is
+ * a new architecture that is more decoupled than the older {@link Model}.
+ *
+ * This class is still experimental. glTF features that are core to 3D Tiles
+ * are supported, but other features such as animation are not yet supported.
+ *
+ * @constructor
+ *
+ * @param {Object} options Object with the following properties:
+ * @param {Uint8Array} [options.gltf] A binary glTF buffer.
+ * @param {Resource|String} [options.basePath=''] The base path that paths in the glTF JSON are relative to.
+ *
+ * @private
+ * @experimental This feature is using part of the 3D Tiles spec that is not final and is subject to change without Cesium's standard deprecation policy.
+ */
+export default function ModelExperimental(options) {
```

Comment:
Another thing, @sanjeetsuhag are we intending to add `.fromGltf()` here or in another PR?

Reply:
I think we should focus on 3D Tiles only for now.